src/config.h

```diff
@@ -112,6 +112,8 @@ typedef struct {
   // Can allow to control the seperation between phrases in different array slots (related to the SLOP parameter in ft.search command)
   // Default value is 100. 0 will not increment (as if all text is a continus phrase).
   unsigned int multiTextOffsetDelta;
+  // bitmap of dialects used by all indices
+  unsigned long long dialects;
```

Comment:
Consider a rename, such as `dialects_usage` or `used_dialects`.

Will we need so many dialects in parallel? (long long)